src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -158,6 +179,101 @@ private static boolean isKillCountFilterInvalid(int varbitValue) {
+            .stream()
+            .map(prop -> prop.substring(prefix.length()))
+            .collect(Collectors.toMap(Function.identity(), key -> configManager.getConfiguration(CONFIG_GROUP, key)));
+        Utils.copyToClipboard(gson.toJson(configMap))
+            .thenRun(() -> plugin.addChatSuccess("Copied current configuration to clipboard"))
+            .exceptionally(e -> {
+                plugin.addChatWarning("Failed to copy config to clipboard");
+                return null;
+            });
+    }
+
+    @Synchronized
+    private void importConfig() {
+        Utils.readClipboard()
+            .thenApplyAsync(json -> {
+                if (json == null) {
+                    plugin.addChatWarning("Clipboard was empty");
+                    return null;
+                }
+
+                try {
+                    return gson.<Map<String, String>>fromJson(json, new TypeToken<Map<String, String>>() {}.getType());
+                } catch (Exception e) {
+                    String warning = "Failed to parse config from clipboard";
+                    log.warn(warning, e);
+                    plugin.addChatWarning(warning);
+                    return null;
+                }
+            })
+            .thenAcceptAsync(this::handleImport)
+            .exceptionally(e -> {
+                plugin.addChatWarning("Failed to read clipboard");
+                return null;
+            });
+    }
+
+    @Synchronized
+    private void handleImport(Map<String, String> map) {
+        if (map == null) return;
+
+        AtomicInteger numUpdated = new AtomicInteger();
+        map.forEach((key, value) -> {
+            String prevValue = configManager.getConfiguration(CONFIG_GROUP, key);
+            String newValue;
+
+            if (key.endsWith("Webhook")) {
+                Collection<String> urls = readDelimited(prevValue).collect(Collectors.toCollection(LinkedHashSet::new));
+
+                long added = readDelimited(value)
+                    .map(urls::add)
+                    .filter(Boolean::booleanValue)
+                    .count();
+
+                if (added > 0) {
+                    newValue = String.join("\n", urls);
+                } else {
+                    newValue = null;
+                }
+            } else if ("ignoredNames".equals(key)) {
+                Collection<String> ignored = readDelimited(config.ignoredNames()).collect(Collectors.toCollection(LinkedHashSet::new));
+                readDelimited(value).forEach(ignored::add);
+
+                if (ignored.size() > this.ignoredNames.size()) {
+                    newValue = String.join("\n", ignored);
+                } else {
+                    newValue = null;
+                }
+            } else {
+                if (Objects.equals(value, prevValue)) {
+                    newValue = null;
+                } else {
+                    newValue = value;
+                }
+            }
```

Comment:
Now that we don't do any sort of processing with the import, do we need to manually handle these keys? My tests of just commenting this out handles things properly still

Reply:
We do need special handling for those fields to merge the new value with the old value instead of just overwriting (e.g., add a new override while keeping the existing one)

Reply:
note that my testing only went so far as to look at the config panel